src/numJewelsInStones.js

```diff
@@ -28,7 +28,26 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  let count = 0;
+  let unique = jewels;
+  const jewelsLength = jewels.length;
+  for (let i = 0; i < jewelsLength; i++) {
+    for (let j = i + 1; j < jewelsLength; j++) {
+      if (unique[i] === unique[j]) {
```

Comment:
why not write?
j this is i + 1
```suggestion
      if (unique[i] === unique[i + 1]) {
```

Reply:
I think [ j ] more clearly, and less one operation

Reply:
it's more expensive.
one loop better than two